app/react/utils/specs/api.spec.js

```diff
@@ -150,6 +172,21 @@ describe('api', () => {
       });
     });
 
+    describe('when request return a server error of too large request', () => {
+      it('should notify the error with this message', async () => {
+        const requestParams = new RequestParams({ key: 'test' }, { header: 'value' });
+        try {
+          await api.post('validation_error', requestParams);
```

Comment:
Is this a self contained thing within the test?  You are making an API call to "validation_error", which is overridden on line 58. I am confused.